.github/DOWNLOADING.md

```diff
@@ -61,38 +68,34 @@ Make sure to set the port to the one you specified in the config.txt,
 and set the Security box to 'Trusted'.
 Then press GO and the server should start up and be ready to join.
 
-### Installation (Linux)
+## Installation (Linux)
 
-The code is fully able to run on linux, however windows is the recommended platform. The code requires 2 libraries, with dependencies below
+The code is fully able to run on linux, however windows is still the recommended platform. The library we use for external functions (RUSTG) requires some extra dependencies.
 
-For MySQL, run the following: `apt-get install libmysqlclient-dev:i386`
+For debian, please download the latest RUSTG release from [https://github.com/ParadiseSS13/rust-g](https://github.com/ParadiseSS13/rust-g), run the following: `apt-get install libssl-dev:i386 pkg-config:i386 zlib1g-dev:i386`.
```

Comment:
For the person that comments "Include CentOS or AWS Linux" instructions. No.

We barely support linux anyways, and debian/ubuntu is the only OS I have enough familiarity with to write these, and I am not about to spin up 20 virtual machines. If youre running CentOS or Arch, you can figure this out.